ProjectNightBot/Commands/GooseCommand.cs

```diff
@@ -0,0 +1,26 @@
+namespace ProjectNightBot.Commands;
+
+using Discord;
+using Discord.WebSocket;
+using SharedInterfaces;
+
+internal class GooseCommand : ICommand
+{
+	public string Name => "goose";
+
+	public SlashCommandProperties Build()
+	{
+		return new SlashCommandBuilder()
+			.WithName(Name)
+			.WithDescription("Get goosed sucker")
+			.Build();
+	}
+
+	public async Task Execute(SocketSlashCommand command)
+	{
+        Random num = new Random();
```

Comment:
You can just use Random.Shared instead of creating a new Random instance every time.